Description: Remove install.man as a dependecy of install. Incorrect patch forwarded upstream caused this
Forwarded: yes
--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@
 
 9wm: 9wm.o event.o manage.o menu.o client.o grab.o cursor.o error.o
 
-install: 9wm install.man
+install: 9wm
 	mkdir -p $(BIN)
 	cp 9wm $(BIN)/9wm
 
